_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,194 @@
+
+The following animation shows a custom animation handled by the Python API retrieving the bones then blending between the neutral pose and a custom walking animation handled through the API:
+
+![pedestrian_animation](/img/posts/2021-11-07/pedestrian.gif){:class="img-fluid rounded mx-auto d-block"}
+
+The following code gets a pedestrian's bones controlling the arms and adjusts the arms' pose:
+
+```py
+# get the bones
+bones = actor.get_bones()
+
+# modify some bones
+new_pose = []
+for bone in bones.bone_transforms:
+    if bone.name == "crl_foreArm__L":
+        bone.relative.rotation.pitch -= 90
+        new_pose.append((bone.name, bone.relative))
+    elif bone.name == "crl_foreArm__R":
+        bone.relative.rotation.pitch -= 90
+        new_pose.append((bone.name, bone.relative))
+
+# set the new pose
+control = carla.WalkerBoneControlIn()
+control.bone_transforms = new_pose
+actor.set_bones(control)
+
+# blend the pose
+actor.blend_pose(0.5)
+```
+
+## New CARLA assets
+
+### New pedestrians
+
+4 new pedestrians have been added to the CARLA assets. 2 adults (each with 3 variants) and 2 children. 
+
+![neew_pedestrians](/img/posts/2021-11-07/new_pedestrians.jpg){:class="img-fluid rounded mx-auto d-block"}
+
+### New volkswagen van
+
+The CARLA garage has built an exciting new vehicle, the __Volkswagen T2 2021__. Add some charm to your city traffic with this beautifully retro styled vehicle. 
+
+![vwvan](/img/posts/2021-11-07/vwvan.gif){:class="img-fluid rounded mx-auto d-block"}
+
+### Opening Doors
+
+All generation 2 vehicles now have articulated doors that can be opened and closed through functions in the python API. This functionality is ideal for simulating scenarios in busy city streets and car parks where doors may open unpredictably while driving past or might be blocking access to parking spaces or passageways. 
+
+![doors](/img/posts/2021-11-07/doors.gif){:class="img-fluid rounded mx-auto d-block"}
+
+## Improved traffic manager logic
+
+The traffic manager has undergone a major update with updated logic for handling vehicle behavior. Traffic now behaves in a more realistic manner at intersections. Vehicle turning is now improved to adapt much better at high speeds. Collision detection at high speeds is enhanced to garner more realistic rapid traffic behavior. 
+
+### User defined paths in traffic manager
+
+The traffic manager has been extended with some great new functionality to extend user control over the behavior of vehicles in the traffic manager. The API now exposes new traffic manager functions that can be used to guide vehicles with a user-defined custom path.
+
+The traffic manager now exposes the `get_next_action` function to access the next action that a traffic manager guided vehicle will take and the `get_all_actions` enables the user to query all possible actions a traffic manager guided vehicle has available.
```

Comment:
traffic manager and action is repeated way too much

Reply:
true dat, I've revised this section